doc/api/process.md

```diff
@@ -1982,6 +1982,19 @@ A boolean value that is `true` if the current Node.js build includes support for
 
 ***
 
+## `process.features.typescript`
+
```

Comment:
This is an experimental feature. It needs to state that it is experimental

Reply:
Does `process.feature.*` count as experimental? I think that this part isn't experimental, but the API is. I don't think it needs to be documented explicitly, I think mentioning that `--experimental-<...>-types` is needed works?

If you disagree, I'll update the doc + emit the warning accordingly.

Reply:
> I think mentioning that `--experimental-<...>-types` is needed works?

That would be wrong, `process.features.typescript` would be defined regardless of what flags are passed IIUC